frontend/src/components/SchedCalendar/CalendarContent/CalendarSched/CalendarSched.js

```diff
@@ -35,14 +38,25 @@ const CalendarSched = (props) => {
     condClass = styles.Filled;
 
     const job = props.timeslot[key];
-    const client = job['client'];
+    client = job['client'];
     clientName = client['name'];
     clientNumber = client['phone'];
     jobAddress = job['address'];
     jobDescription = job['description'];
     jobId = job['jobId'];
-    arrDetails = [{'Id': jobId}, {'Name': clientName}, {'Number': clientNumber},
-      {'Address': jobAddress}];
+    deleteDetails = [
+      {'Id': jobId},
+      {'Name': clientName},
+      {'Number': clientNumber},
+      {'Address': jobAddress},
+    ];
+    editDetails = {
+      'jobId': jobId,
+      'clientId': client['clientId'],
+      'time': key + '',
```

Comment:
What is the reasoning behind concat with '' here?

Reply:
converting to a string value without using the toString() method